opencraft/settings.py

```diff
@@ -951,6 +951,7 @@
 # CORS Settings - https://github.com/adamchainz/django-cors-headers
 CORS_ORIGIN_REGEX_WHITELIST = [
     r"(http|https)://(.*).opencraft.com",
+    r'http://localhost:3000'
```

Comment:
I added this to allow running a local console but there should be a better way, maybe we should try parsing from a JSON env var?

Reply:
@kewne, I can see this below:
```python
if DEBUG:
    CORS_ORIGIN_ALLOW_ALL = True
```

Shouldn't it make local console work if `DEBUG` is `True`?

Reply:
@0x29a I did see that but wasn't sure if we wanted to enable it in `stage` as it might enable insecure features.

Reply:
@kewne oh, I thought you were facing CORS issues with local Ocim. I think you can use local frontend with staging this way:
1. Add `127.0.0.1 local.opencraft.com` to `/etc/hosts`.
2. Run frontend this way: `HOST=local.opencraft.com PORT=80 npm start`.

What do you think?

Reply:
@0x29a This works great, I've reverted my change and added instructions for this to the documentation.